test/integration/order.test.js

```diff
@@ -13,7 +13,7 @@ const describeNoAuthOnly = AUTH_TYPE === 'NONE' ? describe : describe.skip
 
 describeAuthOnly('order - authenticated', function () {
   describe('valid order', function () {
-    this.timeout(3000)
+    this.timeout(5000)
```

Comment:
I'm surprised you need the longer timeout here. Any particular reason so you think?

Reply:
It was actually hitting a timeout